pom.xml

```diff
@@ -61,14 +61,15 @@
   </properties>
 
   <modules>
-    <module>core</module>
-    <module>connector-j-5</module>
-    <module>connector-j-6</module>
-    <module>connector-j-8</module>
-    <module>postgres</module>
-    <module>r2dbc-core</module>
-    <module>r2dbc-mysql</module>
-    <module>r2dbc-postgres</module>
+    <module>jdbc/core</module>
```

Comment:
This is technically the core for both right? 

That said I think these changes should be in two separate PRs: one to reorganize/rename, and one for jdbc-sqlserver

Reply:
I opened #322 to reorganize the folders. Once that's merged, I can rebase this PR.